tests/integration/shop-client-cart-test.js

```diff
@@ -49,9 +49,11 @@ test('it resolves with a new cart on ShopClient#create', function (assert) {
   delete cartAttrs[GUID_KEY];
 
   localStorage.setItem = function (key, value) {
-    assert.ok(key.match(/shopify-buy\.\d+\.\d+/));
-    assert.deepEqual(JSON.parse(value).cart.line_items, cartAttrs.line_items);
-    assert.equal(JSON.parse(value).cart.subtotal_price, cartAttrs.subtotal_price);
+    if (key !== '__storage_test__') {
```

Comment:
so yeah this sucks... the first call to `localStorage` is now the feature test, so these assertions fail the first time.